lib/neat.rb

```diff
@@ -1,4 +1,12 @@
 require "sass"
 require "neat/generator"
 
-Sass.load_paths << File.expand_path("../../core", __FILE__)
+module Neat
+  if defined?(Rails) && defined?(Rails::Engine)
```

Comment:
Do both of these need to be set? Can you have `Rails::Engine` without `Rails`? Or is this so we don't get an error about a missing `Rails` module?

Reply:
```
~% irb
irb(main):001:0> defined?(Rails::Engine)
=> nil
irb(main):002:0> class Rails::Engine
irb(main):003:1> end
NameError: uninitialized constant Rails
	from (irb):2
	from /usr/bin/irb:11:in `<main>'
irb(main):004:0> 
```

Agreeing with Edward here: I think we can just say `defined?(Rails::Engine)`.